frontend/src/pages/privacy.tsx

```diff
@@ -31,23 +31,10 @@ const Privacy = (): JSX.Element => {
           <main>
             <h1>Privacy Policy</h1>
 
-            <p>Last updated: Oct 19, 2021.</p>
+            <p>Last updated: Mar 11, 2022.</p>
```

Comment:
We should retain the October date for Last update and just remove the Summary of key changes section.